docs/core/sdk/file-based-apps.md

```diff
@@ -77,24 +84,18 @@ dotnet file.cs
 
 #### Pass arguments
 
-Pass arguments to your application in several ways:
+Pass arguments to your application by placing them after `--`:
 
 ```dotnetcli
 dotnet run file.cs -- arg1 arg2
 ```
 
-Arguments after `--` are passed to your application. Without `--`, arguments go to the `dotnet run` command:
+Without `--`, arguments go to the `dotnet run` command:
```

Comment:
That's true only for arguments recognized by `dotnet run`, like `--no-build`, other unknown arguments are still forwarded to the app. But I guess that's not necessary to mention and what you have here sounds good enough.